pyramid_oereb/standard/load_sample_data.py

```diff
@@ -174,9 +175,14 @@ def load(self):
         """
         Performs the database operations to load the sample data.
         """
+        # Find data models for each PLR from config
+        plrs = Config._config.get('plrs')
+        models = {plr['code']: plr['source']['params']['models'] for plr in plrs}
+
         from pyramid_oereb.standard.models import contaminated_public_transport_sites, \
-            groundwater_protection_zones, forest_perimeters, motorways_building_lines, \
+            groundwater_protection_zones, motorways_building_lines, \
             contaminated_military_sites
+        forest_perimeters = importlib.import_module(models['ForestPerimeters'])
```

Comment:
Why do we need a change for forest_perimeters, but not the other themes?

Reply:
When we discussed I thought we agreed to do it just for this theme? Should I add other themes and if so which ones? (I think not all test themes have an Oereblex model / not all themes with an Oereblex model are in the test data)

Reply:
To do: use config to load data models + add checks for necessary keys